src/main/resources/log4j.properties

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 ################################################################################
 
-log4j.rootLogger=INFO, console
+log4j.rootLogger=WARN, console
```

Comment:
this silences the info messages when kafka gets a message from up stream. there was so much of it that its very hard to see other output so i turned it off. should we have a problem with kafka that doesnt result in a warning or error message it will be hard to find. let me know if someone thinks this is bad form.